demos/tutorial.md

```
@@ -0,0 +1,390 @@
+This event describes the modifier keys pressed (if any) and the pressed key or rune.
+
+When a Esc key is pressed, an event with its `ev.key.key` set to `Key.esc` is dispatched.
+
+When other key is pressed, it is available as the `Key.graph` of the event.
+
+```d
+switch (ev.key.key)
+{
+case Key.esc:
+    {
+    		info(i"Esc key was pressed");
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 5

(MD010, no-hard-tabs)